assets/js/script.js

```diff
@@ -0,0 +1,194 @@
+//setting up the range of working hours to display in schedule
+const workingHours = [
+  { timeLabel: "8am", key: 8 },
+  { timeLabel: "9am", key: 9 },
+  { timeLabel: "10am", key: 10 },
+  { timeLabel: "11am", key: 11 },
+  { timeLabel: "12pm", key: 12 },
+  { timeLabel: "1pm", key: 13 },
+  { timeLabel: "2pm", key: 14 },
+  { timeLabel: "3pm", key: 15 },
+  { timeLabel: "4pm", key: 16 },
+  { timeLabel: "5pm", key: 17 },
+  { timeLabel: "6pm", key: 18 },
+];
+
+//UTILITY FUNCTIONS
+
+//Get from local storage
+const getFromLS = (key) => {
+  return JSON.parse(localStorage.getItem(key));
+};
+
+//Write to local storage
+const writeToLS = (key, data) => {
+  if (data != "") {
+    localStorage.setItem(key, JSON.stringify(data));
+  }
+};
+
+//Remove item from local storage
+const removeFromLS = (key) => {
+  localStorage.removeItem(key);
+};
+
+//Clear local storage
+const clearLS = () => {
+  localStorage.clear();
+};
+
+//END UTILITY FUNCTIONS
+
+//function to render current date in header (moment.js)
+const renderDate = () => {
+  const currentDate = moment().format("dddd, Do of MMMM YYYY");
+  $("#currentDay").append(currentDate);
+};
+
+//function to handle click on remove buttons
+const handleRemoveClick = (event) => {
+  //gets the data-key for the button clicked
+  const targetKey = $(event.target).attr("data-key");
+  //removes the data-key and its value from local storage
+  removeFromLS(targetKey);
+  //re-renders the scheduler
+  renderScheduler();
+};
+
+//function to handle click on save buttons
+const handleSaveClick = (event) => {
+  //gets the data-key for the button clicked
+  const targetKey = $(event.target).attr("data-key");
+  //gets the value of the corresponding textarea (textarea with same data-key)
+  const taskValue = $(`textarea[data-key=${targetKey}]`).val();
+  //sets the data-key and its value in local storage (for data persistence)
+  writeToLS(targetKey, taskValue);
+};
+
+//function to handle click on the clear button
+const handleClearClick = () => {
+  //calls function to clear local storage
+  clearLS();
+  //re-renders the scheduler
+  renderScheduler();
+};
+
+//function to handle a click in the container - redirects to the right handling function
+const handleClick = (event) => {
+  //stops propagation to other elements in the container
+  event.stopPropagation();
+  //gets the date purpose to know if a save or remove button was clicked
+  const targetPurpose = $(event.target).attr("data-purpose");
+  //if remove button was clicked, then goes to the handleRemoveClick function
+  if (targetPurpose == "remove") {
```

Comment:
Please use `===` instead of `==`
```suggestion
  if (targetPurpose === "remove") {
```